### PR TITLE
Upgrade to 0 10 version

### PR DESCRIPTION
**Pull request by isaacmg410**
This PR upgrades de Participa application to decidim 0.10.1
 

- [x] Upgrade version Decidim to 0.10.1
- [x] add :oc missing locales
- [x] rails decidim:upgrade to add missing migrations
- [x] fix identation to some files
- [x] update the overwritten files

### CODE REVIEW

**Comment by isaacmg410** [comment]
This PR can be reviewed, but it can not be merged into master until the PR #5 has been accepted and merged

File: decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb [hunk 1]
```diff
@@ -73,7 +79,7 @@ def attributes
             # Participa added attributes
 
             #TODO: revisar lo de aplicar el downcase cuando no existe
-            #email: form.email.downcase, 
+            #email: form.email.downcase,
```
Comment (MarcReniu) [inline decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: Can this commented line be erased?

File: decidim-regulations/app/controllers/decidim/participatory_processes/participatory_processes_controller.rb [hunk 2]
```diff
@@ -91,3 +91,104 @@ def is_subcategory(category_id)
     end
   end
 end
+
+#
```
Comment (MarcReniu) [inline decidim-regulations/app/controllers/decidim/participatory_processes/participatory_processes_controller.rb]: ????????

File: decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb (hunk 1 above)
Comment (isaacmg410) [reply decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: Removed

File: decidim-regulations/app/controllers/decidim/participatory_processes/participatory_processes_controller.rb (hunk 2 above)
Comment (isaacmg410) [reply decidim-regulations/app/controllers/decidim/participatory_processes/participatory_processes_controller.rb]: My fault, removed

**Comment by isaacmg410** [comment]
@MarcReniu requested changes applied

**Pull request merged**